notebooks/WFC3/ir_ima_visualization/IR_IMA_Visualization_with_an_Example_of_Time_Variable_Background.ipynb

```diff
@@ -96,7 +96,6 @@
     "- *matplotlib.pyplot* for plotting data\n",
```

Comment:
<p>Also here:</p><p><code>plt.rcParams.update({'font.size': 40})</p><p># Loop starts here</p><p>for ... :</p><p>&nbsp;&nbsp;# Plotting code</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>